guake/tests/test_about.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+import pytest
+
+from guake import guake_version
+from guake.about import AboutDialog
+from locale import gettext as _
+
+
+class TestAboutDialog:
+
+    @classmethod
+    def setup_class(cls):
+        cls.ad = AboutDialog(mock=True)
```

Comment:
So here you need to « mock.patch » the class and not ask it to create the mock for you.

Prefer using « mocker » package (pytest_mock package) it is way more powerful